.github/workflows/firebase-functions-merge.yml

```diff
@@ -0,0 +1,41 @@
+name: Deploy Firebase functions and Firestore rules
```

Comment:
Why are we deploying these two things together, but separately to the build files?
It's probably best to put them in a single 'job'.

Reply:
Sure - I've moved everything into one action